src/ansys/fluent/core/workflow.py

```diff
@@ -366,6 +366,18 @@ def __dir__(self):
             )
         )
 
+    def get_state(self):
+        _state = self._task_container.get_state()
+        if "Workflow" in _state:
+            del _state["Workflow"]
```

Comment:
@prmukherj Isn't it the true intention to `del` everything that's not a `TaskObject` and it just happens that `Workflow` is the only other child of root? 

Also, what you are expressing here is something that will eventually generalise: you're essentially saying that this is a named object container for T=TaskObject.